waterbox/stella/BizhawkInterface.cxx

```diff
@@ -0,0 +1,219 @@
+#include <cstdio>
+#include <cstdint>
+#include "Cart.hxx"
+#include "BizhawkInterface.hxx"
+#include "OSystem.hxx"
+#include "Settings.hxx"
+#include "MediaFactory.hxx"
+#include "Serializer.hxx"
+#include "StateManager.hxx"
+#include "Console.hxx"
+#include "Control.hxx"
+#include "Switches.hxx"
+#include "M6532.hxx"
+#include "TIA.hxx"
+
+uint16_t soundbuffer[4096];

```

Comment:
Can this be placed in the invisible section?

Reply:
Yes, that'd make sense

Reply:
Was this resolved?

Reply:
This array hasn't moved so I'd say no.

Reply:
should be resolved now